### PR TITLE
[WIP]Add pool related cas template and run task

### PR DESCRIPTION
**Pull request by sonasingh46**
This PR will do following :

1. Add pool related cas template and run task artifacts
   to installer.

Signed-off-by: sonasingh46 <sonasingh46@gmail.com>

### CODE REVIEW

File: pkg/install/v1alpha1/cstor_pool_0.7.0.go [hunk 1]
```diff
@@ -37,29 +37,46 @@ metadata:
...
-  # RunNamespace is the namespace where namespaced resources related to pool 
+  # SparseDir is a hostPath directory where to look for sparse files
+  - name: SparseDir
+    value: /var/openebs/sparse
+  # RunNamespace is the namespace where namespaced resources related to pool
   # will be placed
   - name: RunNamespace
-    value: {{.installer.namespace}}
-  taskNamespace: {{.installer.namespace}}
+    value: openebs
```
Comment (avishnu) [inline pkg/install/v1alpha1/cstor_pool_0.7.0.go]: Namespace should be fetched from the install config. Check and change in the required places.

Comment (AmitKumarDas) [reply pkg/install/v1alpha1/cstor_pool_0.7.0.go]: Please check the use of `{{env "NAME_OF_ENV"}}`

File: pkg/install/v1alpha1/cstor_pool_0.7.0.go [hunk 2]
```diff
@@ -74,8 +91,8 @@ spec:
   defaultConfig:
     # RunNamespace is the namespace to use to delete pool resources
   - name: RunNamespace
-    value: {{.installer.namespace}}
-  taskNamespace: {{.installer.namespace}}
+    value: openebs
```
Comment (AmitKumarDas) [inline pkg/install/v1alpha1/cstor_pool_0.7.0.go]: same comments .. please use `env` template function

File: pkg/install/v1alpha1/cstor_pool_0.7.0.go [hunk 3]
```diff
@@ -74,8 +91,8 @@ spec:
...
+  taskNamespace: openebs
```
Comment (AmitKumarDas) [inline pkg/install/v1alpha1/cstor_pool_0.7.0.go]: same comments .. please use `env` template function

**Comment by kmova** [comment]
The changes are being merged with #496

**Pull request closed without merging**